inference/core/devices/utils.py

```diff
@@ -89,7 +89,7 @@ def get_container_id():
     if is_running_in_docker():
         return (
             os.popen(
-                "cat /proc/self/cgroup | grep 'docker' | sed 's/^.*\///' | tail -n1"
+                r"cat /proc/self/cgroup | grep 'docker' | sed 's/^.*\///' | tail -n1"
```

Comment:
I'm not sure if it works as expected. Debian changed behaviour some time ago.

https://stackoverflow.com/questions/69002675/on-debian-11-bullseye-proc-self-cgroup-inside-a-docker-container-does-not-sho